modules/telco-ran-yaml-ref-day-2-operators.adoc

```diff
@@ -76,6 +76,35 @@ include::snippets/ztp_StorageOperGroup.yaml[]
 include::snippets/ztp_StorageSubscription.yaml[]
 ----
 
+[id="ztp-imagebasedupgrade-yaml"]
+.ImageBasedUpgrade.yaml
+[source,yaml]
+----
+include::snippets/ztp_ImageBasedUpgrade.yaml[]
+----
+////
```

Comment:
Commenting LCA snippets out for the moment as I don't know if they should be included